main.yml

```
@@ -534,13 +535,18 @@ landing:
 malware:
   title: Malware
   permissions:
-    method: isInternal
+    - method: apiRequest
+      args:
+        - url: '/api/malware/v1/user-access/access'
+          accessor: 'data'
   api:
     versions:
       - v1
   channel: '#advisor'
   deployment_repo: https://github.com/RedHatInsights/malware-detection-frontend-build
   frontend:
```

you can add a section under here for the isHidden

```suggestion
  frontend:
    isHidden: true
```